src/requests.rs

```diff
@@ -31,43 +31,51 @@ pub struct FeeRates<'f> {
 /// 'listnodes' command
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ListNodes<'f> {
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I hate these. `rust-bitcoincore-rpc` also has them everywhere. There should really be a way to define this on the struct.